### ISSUE TITLE
Switch labeling guide to accordion style

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#2211]
Veronica recently ran some user studies, and one of the big takeaways was that people found the How to Label page overwhelming. Here are some quotes from participants:

>I feel like this could be a little more [clear], it’s a little dense text-wise, I want it to be more focused on questions and the [screenshots] made more compact.

>“It’s really just telling me the basics - [I guess] I really need to look at the tabs [at left]...They do a great job explaining but there’s also a lot to go through. I would want to go [read] through the entire label guide before I started labeling...  “I feel like there is more learning I need to do before I would feel [comfortable continuing to label].

>P4, also, was overwhelmed by the volume of content visible at one time. She suggested it “might be better to accordion questions,” rather than have explanations visible from the start.

I think that last suggestion to switch to an accordion is probably a good idea. The sidebar that lets you navigate to any part of the page mostly accomplishes that, but I can see how having all of the content laid out at once could feel overwhelming.

Thoughts @jonfroehlich ?

**Comment by misaugstad** [comment]
I'm going to consolidate our labeling guide rework into #3282

**Comment by jonfroehlich** [comment]
I don't think we should do any labeling guide work until you and I talk about it.
